### PR TITLE
0.9.18

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: node/src/command.rs [hunk 1]
```diff
@@ -326,6 +326,7 @@ pub fn run() -> Result<()> {
 		// Fix as this run the node
 		None => {
 			let runner = cli.create_runner(&cli.run.normalize())?;
+			let collator_options = cli.run.collator_options();
```
Comment (justinzhou93) [inline node/src/command.rs:329]: i'm assuming there are defaults that come out of this function? cause it doesn't seem like we're adding any specific options here

**Review by justinzhou93 — APPROVED** [review]
i forget, what's the command for formatting the codebase again? mind writing that down in README or doc? i always forget what it is and end up not formatting.

File: node/src/command.rs (hunk 1 above)
Comment (rhuttman) [reply node/src/command.rs:329]: `	/// Create [`CollatorOptions`] representing options only relevant to parachain collator nodes
	pub fn collator_options(&self) -> CollatorOptions {
		CollatorOptions { relay_chain_rpc_url: self.relay_chain_rpc_url.clone() }
	}`

**Review by irsal — COMMENTED** [review]
- Add change log
- Does this change bump any of the runtime versions?

**Review by irsal — APPROVED** [review]

**Pull request merged**